### PR TITLE
changed buttons styles for table

### PR DESCRIPTION
**Pull request by SharedSharkBackup**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1203129252686628

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -46,8 +46,8 @@
     "yup": "^0.32.9"
   },
   "scripts": {
-    "start": "REACT_APP_GIT_SHA=`git rev-parse --short HEAD` react-scripts start",
-    "build": "REACT_APP_GIT_SHA=`git rev-parse --short HEAD` react-scripts build",
```
Comment (SergeyKoliadintsev) [inline package.json:50]: Remove changes in this file!

File: src/levels/admin/components/Orders/index.tsx [hunk 2]
```diff
@@ -113,14 +113,14 @@ const Orders: FC = () => {
         onClick: () => {
           setModalToOpen("downgrade");
         },
-        title: "Downgrade",
+        title: "Downgrade order",
```
Comment (SergeyKoliadintsev) [inline src/levels/admin/components/Orders/index.tsx:116]: Add translate

File: src/levels/admin/components/Orders/index.tsx [hunk 3]
```diff
@@ -113,14 +113,14 @@ const Orders: FC = () => {
...
       },
       {
         id: "new",
         onClick: () => {
           setModalToOpen("add");
         },
-        title: "Order",
+        title: "New order",
```
Comment (SergeyKoliadintsev) [inline src/levels/admin/components/Orders/index.tsx:123]: Add translate

**Review by SharedSharkBackup — COMMENTED** [review]
Added translations

**Pull request merged**